testbeam_analysis/dut_alignment.py

```diff
@@ -858,35 +842,41 @@ def alignment(input_track_candidates_file, input_alignment_file, n_pixels, pixel
         alignment_parameters = _create_alignment_array(n_duts)
         alignment_parameters['translation_z'] = prealignment['z']
 
-        if initial_rotation:
-            if isinstance(initial_rotation[0], Iterable):
-                for dut_index in range(n_duts):
+        beta = np.arctan(prealignment[align_telescope[-1]]["column_c0"] / prealignment[align_telescope[-1]]["z"])
+        alpha = -np.arctan(prealignment[align_telescope[-1]]["row_c0"] / prealignment[align_telescope[-1]]["z"])
```

Comment:
As discussed in https://github.com/SiLab-Bonn/testbeam_analysis/issues/32 it should not be possible to deduce the sign of alpha, beta. This does not matter for start values?